pkg/snapshot/snapshot.go

```diff
@@ -29,6 +27,8 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var snapshotPathPrefix = "/kaniko"
```

Comment:
nit: we could use [KanikoDir](https://github.com/GoogleContainerTools/kaniko/blob/master/pkg/constants/constants.go#L30) instead wherever this is used